app/config/security.yml

```diff
@@ -1,39 +1,13 @@
 security:
-    encoders:
-        Symfony\Component\Security\Core\User\User: plaintext
-
-    role_hierarchy:
-        ROLE_ADMIN:       ROLE_USER
-        ROLE_SUPER_ADMIN: [ROLE_USER, ROLE_ADMIN, ROLE_ALLOWED_TO_SWITCH]
 
     providers:
         in_memory:
-            memory:
-                users:
-                    user:  { password: userpass, roles: [ 'ROLE_USER' ] }
-                    admin: { password: adminpass, roles: [ 'ROLE_ADMIN' ] }
+            memory: ~
 
     firewalls:
         dev:
-            pattern:  ^/(_(profiler|wdt)|css|images|js)/
-            security: false
-
-        login:
-            pattern:  ^/demo/secured/login$
+            pattern: ^/(_(profiler|wdt|error)|css|images|js)/
```

Comment:
It's part of the Twig bundle IIRC, and allows you to check your error
templates, so I'd say yes, we might need it :)

2015-06-11 12:58 GMT+02:00 Christian Flothmann <notifications@github.com>:

> In app/config/security.yml
> <https://github.com/symfony/symfony-standard/pull/818#discussion_r32209761>
> :
>
> >
> >      firewalls:
> >          dev:
> > -            pattern:  ^/(_(profiler|wdt)|css|images|js)/
> > -            security: false
> > -
> > -        login:
> > -            pattern:  ^/demo/secured/login$
> > +            pattern: ^/(_(profiler|wdt|error)|css|images|js)/
>
> Do we really need the |error part here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony/symfony-standard/pull/818/files#r32209761>.
>